app-misc/mc/mc-4.8.20-r2.ebuild

```diff
@@ -13,7 +13,7 @@ SRC_URI="http://ftp.midnight-commander.org/${MY_P}.tar.xz"
 
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x86-solaris"
```

Comment:
removing ~arm?

Reply:
argh, my bad :( Sorry for that. Will push a fix

Reply:
use `ekeyword` to modify an ebuild's `KEYWORDS` to avoid those embarrassing mistakes.

Reply:
Okay thanks. This is my first pull request to main portage tree, so obviously I need to learn some things.